sankeys/services.py

```
@@ -0,0 +1,122 @@
+            # 1. 팀원 데이터 가져오기
+            fs = FinancialStatement.objects.filter(company=company, fiscal_year=year, report_code='11011').first()
+            if not fs or not fs.revenue:
+                logger.warning(f"데이터 없음: {company.company_name}")
+                return None
+            
+            teammate_total_rev = fs.revenue
+            rev_comps = RevenueComposition.objects.filter(company=company, fiscal_year=year)
+
+            # 2. DART 데이터 낚기
+            params = {'crtfc_key': self.api_key, 'corp_code': company.corp_code, 'bsns_year': str(year), 'reprt_code': '11011', 'fs_div': 'CFS'}
+            res = requests.get(self.url, params=params).json()
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)